examples/jsm/animation/CCDIKSolver.js

```diff
@@ -103,6 +105,13 @@ class CCDIKSolver {
 		const links = ik.links;
 		const iteration = ik.iteration !== undefined ? ik.iteration : 1;
 
+		for ( let j = 0; j < links.length; j ++ ) {
+
+			const linkIndex = links[ j ].index;
+			initialQuaternions[ j ] = bones[ linkIndex ].quaternion.clone();
```

Comment:
This method is called for every animation step for each IK bone for all its links. It is unacceptable to use `clone()` in this hot code paths.